### PR TITLE
contrib/gnome-tweaks: update to 46.0

### PR DESCRIPTION
**Pull request by mhmdanas**

### CODE REVIEW

File: contrib/gnome-tweaks/template.py [hunk 1]
```diff
@@ -1,21 +1,40 @@
...
+    "gtk4-devel",
+    "libadwaita-devel",
+    "libgudev-devel",
+    "python-gobject-devel",
+]
 depends = [
-    "gtk+3",
-    "dconf",
-    "mutter",
+    "glib",
```
Comment (q66) [inline contrib/gnome-tweaks/template.py:22]: this is redundant

File: contrib/gnome-tweaks/template.py [hunk 2]
```diff
@@ -1,21 +1,40 @@
...
+    "gnome-desktop",
     "gnome-settings-daemon",
+    "gnome-shell",
+    "gsettings-desktop-schemas",
+    "gtk4",
+    "libadwaita",
+    "libgudev",
```
Comment (q66) [inline contrib/gnome-tweaks/template.py:29]: also redundant

File: contrib/gnome-tweaks/template.py [hunk 3]
```diff
@@ -1,21 +1,40 @@
...
     "libnotify",
-    "libhandy",
+    "pango",
```
Comment (q66) [inline contrib/gnome-tweaks/template.py:31]: ditto

File: contrib/gnome-tweaks/template.py (hunk 2 above)
Comment (mhmdanas) [reply contrib/gnome-tweaks/template.py:29]: i added both this and glib for explicitness but okay i'll remove them

File: contrib/gnome-tweaks/template.py [hunk 4]
```diff
@@ -1,21 +1,37 @@
...
+    "gtk4-devel",
+    "libadwaita-devel",
+    "libgudev-devel",
+    "python-gobject-devel",
+]
 depends = [
-    "gtk+3",
-    "dconf",
-    "mutter",
+    "gnome-desktop",
```
Comment (q66) [inline contrib/gnome-tweaks/template.py:22]: this is also redundant

File: contrib/gnome-tweaks/template.py [hunk 5]
```diff
@@ -1,21 +1,37 @@
...
     "gnome-settings-daemon",
+    "gnome-shell",
+    "gsettings-desktop-schemas",
+    "gtk4",
+    "libadwaita",
     "libnotify",
```
Comment (q66) [inline contrib/gnome-tweaks/template.py:28]: as is libnotify i guess

File: contrib/gnome-tweaks/template.py [hunk 6]
```diff
@@ -1,21 +1,37 @@
...
+]
 depends = [
-    "gtk+3",
-    "dconf",
-    "mutter",
+    "gnome-desktop",
     "gnome-settings-daemon",
+    "gnome-shell",
+    "gsettings-desktop-schemas",
+    "gtk4",
```
Comment (q66) [inline contrib/gnome-tweaks/template.py:26]: as is gtk4

**Pull request merged**